README.md

```diff
@@ -37,6 +37,7 @@ custom:
         swaggerFiles?: ['./doc/endpointFromPlugin.json', './doc/iCannotPutThisInHttpEvent.json', './doc/aDefinitionWithoutTypescript.json']
         swaggerPath?: 'string'
         apiKeyName?: 'string'
+        customApiKeysHeader?: 'string[]'
```

Comment:
Don't know how @SamWSoftware feels about this, but I don't personally like having both of these options that do the same thing essentially. Since it more aligns with what Swagger allows, I'd prefer to go with your implementation of allowing multiple.

Reply:
Thanks for your feedback. I really liked the initiative to automate the API swagger, really very good! About the new field this is a need that I have in some cases to have to send more than one parameter in the header, it may be a need for others too, in this way allowing to pass an array is free to add as many as needed, instead of just one, as it currently is. Please kindly take this into account!

Thank you very much in advance!